client/web/src/Structure/Results Display/State/action.js

```diff
@@ -20,6 +19,7 @@ export const getResultFailure = (payload) => ({
 
 export const getResult = (payload) => (dispatch) => {
   dispatch(getResultRequest())
+  const token = localStorage.getItem("token");
```

Comment:
use enums for token names... i have already created enum folder. Use that.
Also the localstorage methods are in utils, use that